.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# Shared Packages
+packages/ @dev-launchers/platform-enablement
+
+# Ideaspace code
+apps/ideaspace/ @dev-launchers/ideaspace-team
+
+# Recruitment code
+apps/dev-recruiters/ @dev-launchers/dev-recruit-team
+
+# Website code
+apps/website/ @Enjoy2Live @pyxld-kris
+
+# Website code
+apps/app/ @Enjoy2Live @pyxld-kris
+
+# Code in the backend directory
+strapiv4/ @dev-launchers/backend-devs
```

Comment:
do we need to link to the strapi (v3) repo here as well? that is supporting the live backend on production now from what I know and people are still making PRs to it

Reply:
V3 is deprecated and not part of this repo, we're soon going to start consuming data from v4 and change all the requests we make to be compatible with v4 endpoints.

Check out "move-backend" branch which is soon going to be merged to master